core/shared/src/main/scala/scalaz/zio/Ref.scala

```diff
@@ -35,12 +35,6 @@ final class Ref[A] private (private val value: AtomicReference[A]) extends AnyVa
    */
   final def setLater(a: A): IO[Nothing, Unit] = IO.sync(value.lazySet(a))
 
-  /**
```

Comment:
Update the test cases in `RefSpec` where we used `setLater` & `trySet` & `compareAndSet`